projects/wasmer/deterministic.rs

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Consider moving this stuff upstream.
I noticed you never apply for ideal integrations, you know you can make more money by following the steps here:
https://google.github.io/oss-fuzz/advanced-topics/ideal-integration/